feat(releases): Release adoption chart documentation (#3799)
getsentry/sentry-docs: src/docs/product/releases/health/index.mdx

- Session adoption: the number of sessions of a specific release in the last 24 hours.
- User based adoption: the number of users who started the application at least once during the last 24 hours of a specific release.

Adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours.

### Release Adoption Chart

> 
> When a project is selected from the project selecter on the top left of the page, if it has release health set up and if it has releases and user/session data in the selected time period, there will be a chart with the adoption percentages for each release for each data point.
> 
> Adoption data is stored in buckets of 10m, 1h and 1d, the last point on the graph will show the data so far for the last bucket.
> 
> Clicking on a line in the graph you can navigate to that release's [Release Details](/product/releases/health/release-details/) page.
> 
> This feature is applicable only if the project selected is available for mobile or desktop devices.

Reviewer: The project selector is pretty common knowledge for users so we don't need to say where it is and not doing so ensures that we don't have to update this doc if we ever move that selector. I've also changed some of the passive voice to active voice on this.

```suggestion

<Note>

This feature is applicable only if the project selected is available for mobile or desktop devices.

</Note>

If you select a project that has release health configured, and there are releases and user/session data in the selected time period, an adoption chart is displayed. The chart provides you with adoption percentages for each release in the project, over time. 

Each release is represented by a line in the chart, and you can navigate to the [Release Details](/product/releases/health/release-details/) by clicking it.

The chart does not display if you don't select a project or if you select multiple projects.
```

- Session adoption: the number of sessions of a specific release in the last 24 hours.
- User based adoption: the number of users who started the application at least once during the last 24 hours of a specific release.

Adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours.

### Release Adoption Chart

<Note>

This feature is applicable only if the project selected is available for mobile or desktop devices.

</Note>

If you select a project that has release health configured, and there are releases and user/session data in the selected time period, an adoption chart is displayed. The chart provides you with adoption percentages for each release in the project, over time. 

Each release is represented by a line in the chart, and you can navigate to the [Release Details](/product/releases/health/release-details/) by clicking it.

The chart does not display if you don't select a project or if you select multiple projects.

> ![Release Adoption Chart](release_adoption_chart.png)

Reviewer: ```suggestion
![Release Adoption Chart](release_adoption_chart.png)

The chart does not display if you don't select a project or if you select multiple projects.
```

- Session adoption: the number of sessions of a specific release in the last 24 hours.
- User based adoption: the number of users who started the application at least once during the last 24 hours of a specific release.

Adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours.

### Release Adoption Chart

<Note>

This feature is applicable only if the project selected is available for mobile or desktop devices.

</Note>

If you select a project that has release health configured, and there are releases and user/session data in the selected time period, an adoption chart is displayed. The chart provides you with adoption percentages for each release in the project, over time. 

Each release is represented by a line in the chart, and you can navigate to the [Release Details](/product/releases/health/release-details/) by clicking it.

> The chart does not display if you don't select a project or if you select multiple projects.
> 

Reviewer: ```suggestion
```
